packages/@aws-cdk/aws-ec2/README.md

```diff
@@ -1703,3 +1703,24 @@ new ec2.Instance(this, 'Instance1', {
   detailedMonitoring: true,
 });
 ```
+
+## Managed prefix lists
+
+The following demonstrates how to enable [Detailed Monitoring](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/using-cloudwatch-new.html) for an EC2 instance. Keep in mind that Detailed Monitoring results in [additional charges](http://aws.amazon.com/cloudwatch/pricing/).
```

Comment:
Why is a description of detailed monitoring included?

Reply:
I'm sorry. This is the "original description". :)